controllers/queryController.js

```diff
@@ -0,0 +1,92 @@
+const queryModel = require('../models/queryModel');
+const userModel = require('../models/userModel');
+
+const getRecipes = async (req, res) => {
+  const user = req.user;
```

Comment:
acho que aqui um destructuring fica legal
```suggestion
  const { user } = req;
```